### PR TITLE
Release Notes for 1.1.6

### PR DESCRIPTION
**Pull request by JStickler**
Release Notes for CVE release.

Also includes deprecated features for next major release.

### CODE REVIEW

**Comment by JStickler** [comment]
@knrc  and/or @rcernich , could I get a quick review of the Deprecation section please?

File: modules/ossm-rn-deprecated-features.adoc [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+The following custom resources are deprecated in this release and will be removed in a future release.
+
+* `Policy` - The `Policy` resource is deprecated and will be removed in a future Istio release. Migrate to the `PeerAuthentication` resource.
+* `MeshPolicy` - The `MeshPolicy` resource is deprecated and will be removed in a future Istio release. Migrate to the `PeerAuthentication` resource.
+* `v1alpha1` RBAC API -The v1alpha1 RBAC policy is deprecated by the v1beta1 Authorization policy. RBAC (Role Based Access Control) defines `ServiceRole` and `ServiceRoleBinding` objects.
+** `ServiceRole`
+** `ServiceRoleBinding`
+* `RbacConfig` - `RbacConfig` implements the `ClusterRbacConfig` Custom Resource Definition for controlling Istio RBAC behavior.
+** `ClusterRbacConfig`
+** `ServiceMeshRbacPolicy`
```
Comment (knrc) [inline modules/ossm-rn-deprecated-features.adoc:27]: ServiceMeshRbacConfig. I'm not sure what this part is intending though, ClusterRbacConfig was replaced by ServiceMeshRbacConfig in SM 1.0

**Review by knrc — CHANGES_REQUESTED** [review]

File: modules/ossm-rn-new-features.adoc [hunk 2]
```diff
@@ -65,7 +69,11 @@ This release of {ProductName} adds support for a disconnected installation.
 == New features {ProductName} 1.1.0
 
 This release of {ProductName} adds support for Istio 1.4.6 and Jaeger 1.17.1.
+////
+== New features {ProductName} 1.0.12
 
+This release of {ProductName} addresses Common Vulnerabilities and Exposures (CVEs) and bug fixes.
+////
 == New features {ProductName} 1.0.11
```
[the remark this answers is not in the document]
Comment (JStickler) [reply modules/ossm-rn-new-features.adoc:77]: I commented it out after this week's program call.

[the remark this answers is not in the document]
Comment (knrc) [reply modules/ossm-rn-new-features.adoc:77]: Ah my bad, sorry about that

File: modules/ossm-rn-deprecated-features.adoc [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+= Deprecated features
+Some features available in previous releases have been deprecated or removed.
+
+Deprecated functionality is still included in {product-title} and continues to be supported; however, it will be removed in a future release of this product and is not recommended for new deployments.
+
+== Deprecated features {ProductName} 1.1.5
+
+The following custom resources are deprecated in this release and will be removed in a future release.
+
+* `Policy` - The `Policy` resource is deprecated and will be removed in a future Istio release. Migrate to the `PeerAuthentication` resource.
```
[the remark this answers is not in the document]
Comment (JStickler) [reply modules/ossm-rn-deprecated-features.adoc:20]: Done (and this is why I wanted a review, thank you!)

File: modules/ossm-rn-deprecated-features.adoc (hunk 1 above)
Comment (JStickler) [reply modules/ossm-rn-deprecated-features.adoc:27]: That was listed on the Google doc from when we were discussing differences between 1.4 and 1.6.  Should I pull it completely or should it be revised somehow?

From that doc:
"Deprecated: v1alpha1 RBAC API (had been deprecated in 1.4 already, replaced by  v1beta1 AuthorizationPolicy)
ServiceRole
ServiceRoleBinding
RbacConfig
ClusterRbacConfig (ServiceMeshRbacPolicy)"

File: modules/ossm-rn-deprecated-features.adoc (hunk 3 above)
[the remark this answers is not in the document]
Comment (JStickler) [reply modules/ossm-rn-deprecated-features.adoc]: done

File: modules/ossm-rn-deprecated-features.adoc (hunk 1 above)
Comment (JStickler) [reply modules/ossm-rn-deprecated-features.adoc]: @knrc and/or @rcernich , please check updated text here for me.

**Review by neal-timpe — CHANGES_REQUESTED** [review]
Can you update the `:ProductVersion:` to 1.1.7 in modules/ossm-document-attributes.adoc.

File: modules/ossm-rn-deprecated-features.adoc [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+* `MeshPolicy` - The `MeshPolicy` resource is deprecated and will be replaced by the `PeerAuthentication` resource in a future release.
+* `v1alpha1` RBAC API -The v1alpha1 RBAC policy is deprecated by the v1beta1 `AuthorizationPolicy`. RBAC (Role Based Access Control) defines `ServiceRole` and `ServiceRoleBinding` objects.
+** `ServiceRole`
+** `ServiceRoleBinding`
+* `RbacConfig` - `RbacConfig` implements the Custom Resource Definition for controlling Istio RBAC behavior.
+** `ClusterRbacConfig`(versions prior to {ProductName} 1.0)
+** `ServiceMeshRbacConfig` ({ProductName} version 1.0 and later)
+* In Kiali, the `login` and `LDAP` strategies are deprecated. A future version will introduce authentication using OpenID providers.
+
+The following components are are deprecated in this release and will be replaced by the *Istiod* component in a future release.
```
Comment (knrc) [inline modules/ossm-rn-deprecated-features.adoc:30]: are also deprecated

**Review by knrc — CHANGES_REQUESTED** [review]
Minor typo to update, otherwise lgtm

File: modules/ossm-rn-deprecated-features.adoc (hunk 4 above)
Comment (JStickler) [reply modules/ossm-rn-deprecated-features.adoc]: Fixed

**Review by neal-timpe — APPROVED** [review]

**Review by knrc — APPROVED** [review]

**Pull request merged**